pallets/salp/src/lib.rs

```diff
@@ -779,6 +781,63 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[pallet::weight(T::WeightInfo::refund())]
+		#[transactional]
+		pub fn batch_refund(
+			origin: OriginFor<T>,
+			#[pallet::compact] index: ParaId,
+		) -> DispatchResult {
+			ensure_signed(origin)?;
+
+			let mut fund = Self::funds(index).ok_or(Error::<T>::InvalidParaId)?;
+			ensure!(fund.status == FundStatus::Failed, Error::<T>::InvalidFundStatus);
```

Comment:
in batch_refund function, the fund.status is required to be FundStatus::Failed, while in the refund function, the fund status is required to be FundStatus::RefundWithdrew. Is there any difference between these two functions in regard to their execution contexts?